docs/en/latest/how-to-build.md

```diff
@@ -94,6 +95,23 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
   make install
   ```
 
+## Step 2: Install ETCD
+
+This step is required when installing Apache APISIX via RPM or Docker.
+
+You can install ETCD via Docker or binary etc. The following command installs ETCD via binary.
+
+```shell
+ETCD_VERSION='3.4.13'
+wget https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz
+tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
+    cd etcd-v${ETCD_VERSION}-linux-amd64 && \
+    sudo cp -a etcd etcdctl /usr/bin/
+nohup etcd &
+```
+
+Note: When ETCD is started with the command `nohup etcd &`, the data in ETCD is not stored persistently. For more information on ETCD, please refer to the [ETCD documentation](https://etcd.io/docs/).
```

Comment:
`When ETCD is started with the command `nohup etcd &`, the data in ETCD is not stored persistently.` This statement is incorrect. We can remove it as this is not a doc about etcd.

Reply:
done